composer.json

```diff
@@ -104,6 +104,9 @@
     "extra": {
         "branch-alias": {
             "dev-master": "1.10-dev"
+        },
+        "symfony": {
+            "require": "^6.4"
```

Comment:
it allow to fix the docker dev build

Reply:
With that configuration, using docker, it will set up dependencies with Symfony 6.4